lib/services/version_service.dart

```diff
@@ -138,6 +142,9 @@ class VersionService {
       } else if (int.parse(latestPackageNumbers[2]) >
           int.parse(currentPackageNumbers[2])) {
         isNewVersionAvailable = true;
+      } else if (int.parse(latestPackageNumbers[3]) >
+          int.parse(packageInfo.buildNumber)) {
+        isNewVersionAvailable = true;
```

Comment:
Does the array have 4 fields in all platforms or only in windows ?

Reply:
@sachins-geekyants we're missing the comparison between the current and latest build numbers and we don't have the latest Windows version number in the response to check so that the request update dialog always shows in Windows build